dpdispatcher/submission.py

```diff
@@ -169,11 +169,16 @@ def run_submission(self, *, exit_on_submit=False, clean=True):
             self.check_all_finished()
             self.handle_unexpected_submission_state()
 
+        ratio_failure = self.resources.kwargs.get('ratio_failure', 0)
```

Comment:
Why is it called `ratio_failure`? Isn't it `ratio_unfinished`?

Reply:
`ratio_unfinished` is okay for me. I use `ratio_failure` since I saw there is a same param in dpgen, which accomplishes the same thing:
https://github.com/deepmodeling/dpgen/blob/master/dpgen/dispatcher/DispatcherList.py#L48